packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -211,6 +213,9 @@ function BaseLightningElementConstructor(this: LightningElement): LightningEleme
         vm.getHook = getHook;
     }
 
+    // Making the component instance a live value when using Locker to support expandos.
+    defineProperty(component, tagPropertyKey, EmptyObject);
```

Comment:
Can we add a test for this and the reactive property? Checking if the instance has the `Symbol.for('@@lockerLiveValue')`.